index.html

```diff
@@ -2,9 +2,118 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Девайс</title>
+    <title>device</title>
   </head>
+
   <body>
+    <header class="page-header">
+      <a class="logo" href=img alt="logo-device" src="#"></a>
+      <form class="search">
+        <input type="search" value="Текст поиска">
+        <button type="submit">Найти</button>
+      </form>
+      <section class="next-menu">
+        <ul class="button-menu"></ul>
+        <li class="button-menu-icon"><a href="#" class="button">Войти</a></li>
+        <li class="button-menu-icon"><a href="#" class="button">Сравнить</a></li>
+        <li class="button-menu-icon"><a href="#" class="button">Корзина</a></li>
+        </ul>
+      </section>
+      <nav class="navigation">
+        <ul class="main menu">
+          <li class="main-menu-item"><a class="main-menu-link" href="#">"Каталог товаров"</a></li>
+          <li class="main-menu-item"><a class="main-menu-link" href="#"></a>Доставка</li>
+          <li class="main-menu-item"><a class="main-menu-link" href="#"></a>Гарантия</li>
+          <li class="main-menu-item"><a class="main-menu-link" href="#"></a>Контакты</li>
+        </ul>
+      </nav>
+      </header>
+      <main class="main-index">
+          <img src="copter.jpg">
+          <h1 class="visually-hidden">Порхает как бабочка,жалит как пчела!</h1>
+          <p class="advantages-description">Этот обычный,на первый взгляд, квадрокоптер оснащен мощным лазарем,замаскированным под обычную камеру.</p>
+          <a class="podrobnee" href="#">Подробнее</a>
+          <p>800м 50м</p>
+          <p>Дальность полета Радиус поражения</p>
+          <section class="catalog">
+          <ul class="catalog-product">
+          <li class="catalog-item"><a href="#"> Виртуальная реальность</li>
+          <li class="catalog-item"><a href="#"> Моноподы для селфи</li>
+          <li class="catalog-item"><a href="#"> Экшн камеры</li>
+          <li class="catalog-item"><a href="#"> Фитнес-браслеты</li>
+          <li class="catalog-item"><a href="#"> Умные часы</li>
+          <li class="catalog-item"><a href="#"> Квадрокоптеры</a></li>
+          </ul>
+          </section>
+          <section class="delivery">
+            <div>
+            <h2 class="delivery"">Доставка</h2>
+            <p class="delivery-text">Мы с удовольствием доставим ваш товар прямо к вашему подъезду совершенно бесплатно!Ведь мы неплохо заработаем,поднимая его на ваш этаж.</p>
+            <ul class="delivery-advantages">
+              <li class="delivery-item"> <a href="#"> Доставка </a></li>
+              <li class="delivery-item"> <a href="#"> Гарантия </a></li>
+              <li class="delivery-item"> <a href="#"> Кредит </a></li>
+            </ul>
+            </div>
```

Comment:
Див закрываешь перед ul-кой.
```suggestion
            </div>
            <ul class="delivery-advantages">
              <li class="delivery-item"> <a href="#"> Доставка </a></li>
              <li class="delivery-item"> <a href="#"> Гарантия </a></li>
              <li class="delivery-item"> <a href="#"> Кредит </a></li>
            </ul>
```